feat: add Standalone system [xi]

wescopeland opened the pull request:
Adds a new system ID, `102`, for Standalone.

To insert into the prod DB on deploy:

```sql
INSERT INTO Console (ID, Name) VALUES (102, 'Standalone');
```

Jamiras commented on app/Platform/Commands/AddStandaloneSystem.php:
```diff
@@ -0,0 +1,67 @@
+            ]);
+        }
+    }
+}
```
Wouldn't it be better to do this with a migration?

Is there going to be another ID other than 102?

wescopeland replied:
To the best of my knowledge, none of our migrations write rows to the DB. I vaguely recollect a conversation that happened on another PR where we decided not to take that approach, though I can't remember the PR anymore.

As I understand our implementation plan, the new system will only have a single system ID.

Jamiras replied:
You're thinking about the Messages PR. That was a hefty chunk of logic that had to be executed. This is just inserting a single row into the DB. For previous new systems, that's always just been done by hand some time before deploying the code.

wescopeland replied:
Migration added in latest and command removed.

Jamiras commented on config/systems.php:
```diff
@@ -578,4 +578,11 @@
+        'name_short' => 'Indie',
+        'manufacturer' => '',
+        'active' => false,
+    ],
```
This causes the tooltip to show Standalones (maybe it should be singular)

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/04a0e01d-1960-4068-aac2-18b242ff4a7e)

and Indie is shown in the recent games list.

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/78e68077-1151-4f7a-a2b8-8ac8718651b4)

Also, I'm not sold on 'Indie' being the short form of Standalones, but I don't have any better ideas at this time.

wescopeland replied:
Changed the short name to "N/A" and set the full name to singular "Standalone".

wescopeland commented on database/migrations/2024_01_03_000000_add_standalone_system.php:
```diff
@@ -0,0 +1,57 @@
+return new class() extends Migration {
+    public function up(): void
+    {
+        if (App::environment('testing')) {
+            return;
+        }
```
I must confess I'm really not in love with this.

However, there are numerous tests that rely on only a certain number of console IDs existing in the test database.

Part of me still feels like it'd be better if this write operation were not in a migration.

Jamiras replied:
I think we should just craft an insert statement and run it directly against the prod DB.

As long as the seeder populates it for general use, any tests relying on this specific console can add it themselves.

wescopeland commented:
I think this should be all set.

* I've updated the OP with a crafted SQL statement to add the console to prod.
* I've added a commit co-authored by @oddgoo to include the new Standalone console icon (please give some credit to oddgoo when crafting future release notes).
* I've removed the migration that was previously in the PR.

Jamiras commented:
I've run the command on the prod and stage DBs.
